### ISSUE TITLE
\GoogleUrl has colliding constructor definitions coming from traits in 

### ISSUE THREAD

**Issue by olypros** [serp-spider/documentation#11]
**Fatal error: **
```
Serps\SearchEngine\Google\GoogleUrl has colliding constructor definitions coming from traits in 

...\vendor\serps\search-engine-google\src\GoogleUrl.php on line 123
```

**Comment by gsouf** [comment]
Hi @ricking06 

Please can you tell me what version of Serps you are running?

**Comment by olypros** [comment]
"serps/core":"@dev",
        "serps/http-client-curl":"@dev",
        "serps/search-engine-google":"@dev",

**Comment by gsouf** [comment]
Is it also happening with v 0.1 ?

**Comment by olypros** [comment]
it wasnt happening with v0.1

**Comment by gsouf** [comment]
Closing this, becuase also repported in serp-spider/search-engine-google#50 that is a better place for this issue

**Comment by gsouf** [comment]
@ricking06 it's fixed by serp-spider/search-engine-google#50 

However I recommand you to not use version 0.2 for the moment because it's still a development version.
